torch/nn/utils/rnn.py

```diff
@@ -25,62 +25,69 @@ class PackedSequence(PackedSequence_):
 
     Attributes:
         data (Variable): Variable containing packed sequence
-        batch_sizes (list[int]): list of integers holding information about
-            the batch size at each sequence step
+        batch_sizes_variable (Variable): list of integers holding
+            information about the batch size at each sequence step
+        batch_sizes (list of int): equivalent to
```

Comment:
Eh I think I'm ok with making this always a tensor. This way gives us backward compatibility, but:
- `.batch_sizes` access suddenly becomes expensive
- we never want people to use `.batch_sizes_variable` because it's very verbose

Reply:
ok, I will do the simplification